gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -211,9 +210,11 @@ def run_compute_info(mt: hl.MatrixTable) -> hl.Table:
             "release": mt.meta.release,
             "unrelated": ~mt.meta.sample_filters.relatedness_filters.related,
         },
+        n_partitions=None,
     )
+    info_ht = info_ht.checkpoint(hl.utils.new_temp_file("compute_info"), overwrite=True)
```

Comment:
I didn't know this (`new_temp_file`) existed, that is so cool. do you need to specify "ht" here also?